src/pages/HistoryPage/Notes.tsx

```diff
@@ -24,7 +24,10 @@ export const NotesSection = ({
 }) => {
   const receivedLength = received.filter(g => g.gift_private?.note).length;
   const sentLength = sent?.filter(g => g.gift_private?.note).length;
-  const epochStatementsLength = epochStatements?.length;
+  const filteredEpochStatements = epochStatements?.filter(
+    e => e.bio && e.bio.length > 0
```

Comment:
I would have actually done this on the DB side in the query. Like this: (untested)

https://github.com/coordinape/coordinape/compare/filter-epoch-statement-list?expand=1